syntax = "proto3";

package packet;

message DisconnectPacket {
	string reason = 1;
}